Source/WebKit/Shared/API/glib/WebKitURIRequest.cpp

```diff
@@ -32,7 +32,7 @@ enum {
     N_PROPERTIES,
 };
 
-WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
+WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN // GTK port
```

Comment:
I'll stop mentioning these. Basically unless the file ends in "Gtk.cpp" or is in a gtk/ directory, it's probably shared by WPE as well.